src/Setup/DevDivVsix/PortableFacades/PortableFacades.swr

```diff
@@ -1,26 +1,28 @@
 use vs
 
 package name=PortableFacades
-        version=1.5.0.0
+        version=1.6.0.0
 
 folder InstallDir:\Common7\IDE\PrivateAssemblies
+    file source="$(NuGetPackageRoot)\Microsoft.Win32.Primitives\4.3.0\lib\net46\Microsoft.Win32.Primitives.dll" vs.file.ngen=yes
     file source="$(NuGetPackageRoot)\System.AppContext\4.3.0\lib\net46\System.AppContext.dll" vs.file.ngen=yes
     file source="$(NuGetPackageRoot)\System.Console\4.3.0\lib\net46\System.Console.dll" vs.file.ngen=yes
     file source="$(NuGetPackageRoot)\System.Diagnostics.FileVersionInfo\4.3.0\runtimes\win\lib\net46\System.Diagnostics.FileVersionInfo.dll" vs.file.ngen=yes
     file source="$(NuGetPackageRoot)\System.Diagnostics.Process\4.3.0\runtimes\win\lib\net46\System.Diagnostics.Process.dll" vs.file.ngen=yes
     file source="$(NuGetPackageRoot)\System.Diagnostics.StackTrace\4.3.0\lib\net46\System.Diagnostics.StackTrace.dll" vs.file.ngen=yes
     file source="$(NuGetPackageRoot)\System.IO.Compression\4.3.0\runtimes\win\lib\net46\System.IO.Compression.dll" vs.file.ngen=yes
     file source="$(NuGetPackageRoot)\System.IO.FileSystem\4.3.0\lib\net46\System.IO.FileSystem.dll" vs.file.ngen=yes
+    file source="$(NuGetPackageRoot)\System.IO.FileSystem.DriveInfo\4.3.0\runtimes\win\lib\net46\System.IO.FileSystem.DriveInfo.dll" vs.file.ngen=yes
     file source="$(NuGetPackageRoot)\System.IO.FileSystem.Primitives\4.3.0\lib\net46\System.IO.FileSystem.Primitives.dll" vs.file.ngen=yes
     file source="$(NuGetPackageRoot)\System.IO.Pipes\4.3.0\runtimes\win\lib\net46\System.IO.Pipes.dll" vs.file.ngen=yes
-    file source="$(NuGetPackageRoot)\System.Net.Security\4.3.0\runtimes\win\lib\net46\System.Net.Security.dll" vs.file.ngen=yes
+    file source="$(NuGetPackageRoot)\System.Net.Security\4.3.0\lib\net46\System.Net.Security.dll" vs.file.ngen=yes
```

Comment:
@ericstj We're looking at the output of a project.assets.json and seeing that for the runtime facades for targeting 4.6 we're seeing the package contains both stuff in lib and stuff in runtimes. We're not sure why the tooling is changing this, but can't explain why the package has both files either. Are they the same? Interchangeable?